src/non-standalone/Dialog/Types.ts

```diff
@@ -170,3 +171,66 @@ export interface IDialogConfigSign {
 	 */
 	setSignature?: (url: string) => void;
 }
+
+/**
+ * Object properties for image
+ */
+export interface ImageSelectorBoxProps {
+	/**
+	 * The base64 string of signature
+	 */
+	src: string;
+	/**
+	 * Boolean flag to set primary image
+	 */
+	setPrimary: boolean;
+}
+/**
+ * Props which will be provided in image box dialog
+ */
+export interface IDialogConfigImageBox {
```

Comment:
all non dialog specific props should be moved to the props interface of the standalone component you should create for the dialog contents. then extend the standalone component's props interface here